src/provisioner/log.py

```diff
@@ -16,32 +16,68 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 
 import os
+
 from cortx.utils.log import Log
+from cortx.provisioner import const
 
 
 class CortxProvisionerLog(Log):
     """ Redirect log message to log file and console using cortx utils logger """
 
+    default_log_path = '/var/log/cortx'
+    app_name = 'provisioner'
     logger = None
 
     @staticmethod
-    def init(service_name, log_path=None, level='INFO', console_output=False):
+    def initialize(service_name, log_path=None, level='INFO', console_output=False):
```

Comment:
Why set log_path as default in function definition itself...

Reply:
Now changed this to `log_path=const.DEFAULT_LOG_PATH`.